packages/ui-components/package.json

```diff
@@ -67,8 +70,8 @@
     "jest-environment-jsdom": "29.2.0",
     "jest-runner-eslint": "1.1.0",
     "prettier": "2.7.1",
-    "react": "17.0.2",
-    "react-dom": "17.0.2",
+    "react": "^18.2.0",
```

Comment:
@lassenordahl it looks like there are some dependencies listed twice.

Reply:
I'll remove it from `devDependencies`

-- https://stackoverflow.com/questions/18875674/whats-the-difference-between-dependencies-devdependencies-and-peerdependencies